app/Providers/RouteServiceProvider.php

```diff
@@ -92,9 +96,23 @@ protected function mapServiceRoutes()
      */
     protected function mapDataRoutes()
     {
-        Route::prefix('data')
+        Route::domain('arcv-service.test')
+            ->prefix('data')
             ->middleware(['web', 'isNotProduction'])
             ->namespace($this->namespace . '\Service')
             ->group(base_path('routes/data.php'));
     }
+
+    /**
+     * Define the "service" routes for the application.
+     *
+     * @return void
+     */
+    protected function mapStoreRoutes()
+    {
+        Route::domain('arcv-store.test')
+            ->middleware(['web'])
```

Comment:
Just checking still middleware web, right?

Reply:
Yes, the web *middleware* was identical between the installs. 
Saw no reason to change it.